src/hotspot/share/cds/cdsConfig.cpp

```diff
@@ -467,10 +522,26 @@ void CDSConfig::check_aotmode_record() {
 }
 
 void CDSConfig::check_aotmode_create() {
-  if (FLAG_IS_DEFAULT(AOTCache)) {
-    vm_exit_during_initialization("AOTCache must be specified when using -XX:AOTMode=create");
+  if (FLAG_IS_DEFAULT(AOTConfiguration)) {
+    vm_exit_during_initialization("-XX:AOTMode=create cannot be used without setting AOTConfiguration");
```

Comment:
Same suggestion regarding error msg:
Replace `-XX:AOTMode=create cannot be used without setting AOTConfiguration` 
with 
`AOTConfiguration must be specified when using -XX:AOTMode=create`

Reply:
Fixed.